chain/rosetta-rpc/src/utils.rs

```diff
@@ -274,9 +274,14 @@ fn get_liquid_balance_for_storage(
     account: &near_primitives::account::Account,
     storage_amount_per_byte: near_primitives::types::Balance,
 ) -> near_primitives::types::Balance {
-    let required_amount =
-        near_primitives::types::Balance::from(account.storage_usage()) * storage_amount_per_byte;
-    required_amount.saturating_sub(account.locked())
+    let staked_for_storage = if account.storage_usage() <= 770 {
```

Comment:
nit: instead of using a magic 770, maybe make it named constant or maybe create a function `is_zero_balance_account`?

(Assuming a dependency on `node-runtime` is not an option, otherwise you could important `ZERO_BALANCE_ACCOUNT_STORAGE_LIMIT` or even make `is_zero_balance_account` pub and import that. This could prevent similar errors in the future.)